rhel/SPECS/dbaas-tools.spec

```diff
@@ -0,0 +1,60 @@
+%define debug_package %{nil}
+
+%global commit_aws          d7c0b2e9131faabb2b09dd804a35ee03822f8447
+%global shortcommit_aws     %(c=%{commit_aws}; echo ${c:0:7})
+
+%global install_golang 1
+
+%define build_timestamp %(date -u +"%y%m%d%H%M")
+%define release         1
+%define rpm_release     %{release}.%{build_timestamp}%{?dist}
+
+Name:           dbaas-tools
+Version:        0.5.1
+Release:        %{rpm_release}
+Summary:        A set of tools for Percona DBaaS
+License:        ASL 2.0
+URL:            https://github.com/kubernetes-sigs/aws-iam-authenticator
+Source0:        https://github.com/kubernetes-sigs/aws-iam-authenticator/archive/%{commit_aws}/aws-iam-authenticator-%{shortcommit_aws}.tar.gz
+
+%if %{install_golang}
+BuildRequires:   golang >= 1.13.0
+%endif
+
+%description
+%{summary}
+
+%description
+%{summary}
+
+
+%prep
+%setup -T -c -n aws-iam-authenticator-%{commit_aws}
+%setup -q -c -a 0 -n aws-iam-authenticator-%{commit_aws}
+mkdir -p src/github.com/kubernetes-sigs/
+mv aws-iam-authenticator-%{commit_aws} src/github.com/kubernetes-sigs/aws-iam-authenticator-%{commit_aws}
+
+
+%build
+export GOPATH="$(pwd)"
+export CGO_ENABLED=0
+export USER=builder
+
+cd src/github.com/kubernetes-sigs/aws-iam-authenticator-%{commit_aws}
+sed -i '/dockers:/,+35d' .goreleaser.yaml
```

Comment:
why do we need this?

Reply:
`.goreleaser.yaml` includes build of dockers, we don't need their dockers, and goreleaser doesn't support skipping them by setting a flag, so I had to remove them by sed